tests/databases/api_tests.py

```diff
@@ -347,6 +347,7 @@ def test_create_database_fail_sqllite(self):
         """
         Database API: Test create fail with sqllite
         """
+        app.config["PREVENT_UNSAFE_DB_CONNECTIONS"] = True
```

Comment:
To make these even more resilient to flakery, I believe patching the dict would be the best option here, like here:
https://github.com/apache/incubator-superset/blob/164db3e5a13c21137afb56a3044ef3f1aaf89e11/tests/charts/api_tests.py#L1108-L1110

Reply:
Sure, good idea. Done